script/build_demo_file.sh

```diff
@@ -7,13 +7,14 @@ cp -R visualization/dist/webpack/ gh-pages/visualization/app/
 analysis/gradlew -p analysis/ installDist
 
 mkdir gh-pages/demo_files
-cd gh-pages/demo_files
+cd gh-pages/demo_files || exit
 CCSH=../../analysis/build/install/codecharta-analysis/bin/ccsh
 
 git log --numstat --raw --topo-order --reverse -m > git.log
 git ls-files > file-name-list.txt
 
-$CCSH gitlogparser git.log -o codecharta_git.cc.json -n file-name-list.txt -nc
+# Data for for both visualization and analysis
+$CCSH gitlogparser log-scan --git-log git.log --repo-files file-name-list.txt -o codecharta_git.cc.json -nc
```

Comment:
Couldn't we use the simpler repo-scan command. Therefore, we do not need to maintain the commands used for generating the git log and the file list.

Reply:
Yes, that would definetely work, this change is more of a "drop in" fix with the minimal possible changes to not disturb anything else. Additionally, doing the git-log and git-ls calculations like this in the shell is a bit faster.